components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/drop-elements.js

```diff
@@ -1466,6 +1503,10 @@ define(['require', 'log', 'lodash', 'jquery', 'partition', 'stream', 'query', 'f
             });
 
             newElement.on('keydown', function (key) {
+                if ( key.keyCode == 46 || key.keyCode == 8 ) {
```

Comment:
shall we introduce a constant for the 46, 8, 13, 9, 39, 37 and 27?

Reply:
Assign constants to keyCodes